.bazelrc

```diff
@@ -1,3 +1,5 @@
-build --cxxopt=-std=c++17
+build --enable_platform_specific_config
+build:linux --cxxopt=-std=c++17
+build:windows --cxxopt=-std:c++17
```

Comment:
@Shikugawa I think we should set up CI to keep compiling checks in the future, what do you think?

Reply:
Yes. We should prepare for windows container on github actions in the future.

Reply:
Let's add an issue to track?